agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -785,9 +785,9 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
             LOG(ERROR) << "Failed building message!";
             return false;
         }
-        auto tuple_supported_channels = response->supported_channels_list(0);
-        copy_radio_channels(ap_wlan_hal->get_radio_info().supported_channels,
-                            &std::get<1>(tuple_supported_channels));
+        auto tuple_preferred_channels = response->supported_channels_list(0);
+        copy_radio_channels(ap_wlan_hal->get_radio_info().preferred_channels,
+                            &std::get<1>(tuple_preferred_channels));
```

Comment:
same here
`preferred_channels = supported_channels` is confusing
IMO you need to use preferred_channels in all ACS related CMDUs  as a preparation commit.

Reply:
Changed all ACS related variables and CMDUs to use `preferred_channels`